src/content/developers/docs/networks/index.md

```diff
@@ -52,7 +50,7 @@ A proof-of-work testnet; this means it's the best like-for-like representation o
 
 #### Goerli {#goerli}
```

Comment:
Let's put Goerli top of this list if it is expected to be the main testnet people use.